rs/sns/governance/src/types.rs

```diff
@@ -2304,8 +2304,10 @@ impl NeuronRecipe {
 
     // TODO[NNS1-3676]: Provide a proper implementation for this function once new SNSs are
     // TODO[NNS1-3676]: to begin using topics-based following.
-    pub(crate) fn construct_topic_followees(&self) -> BTreeMap<u64, Followees> {
-        btreemap! {}
+    pub(crate) fn construct_topic_followees(&self) -> TopicFollowees {
```

Comment:
NCR

I wish we would converge towards `new` being the usual name for "make a new thing" functions. Certainly, when the name is just one word `new` is pretty standard.

Reply:
I also don't like the name of these functions. Following the local convention here; let's see if it's worth refactoring outside of this PR.